numpy/random/tests/test_random.py

```diff
@@ -459,6 +459,10 @@ def test_choice_nan_probabilities(self):
         p = [None, None, None]
         assert_raises(ValueError, np.random.choice, a, p=p)
 
+    def test_choice_probas_not_contiguous(self):
+        p = np.repeat(np.array([[0.1, 0, 0.3, 0.6, 0]]).T, 3, axis=1)                        
+        random.choice(5, 3, p=p[:, 1])
+
```

Comment:
test should also try `np.random.Generator().choice(...)` which would also fail.

Reply:
... and `np.random.multinomial`, and `np.random.Generator().multinomial`

Reply:
This is in the wrong file.  Should be `test_randomstate.py`

Reply:
And again in `test_generator_mt19937.py`